src/main/java/at/hannibal2/skyhanni/features/garden/GardenOptimalSpeed.kt

```diff
@@ -52,22 +42,16 @@ class GardenOptimalSpeed {
 
     @SubscribeEvent
     fun onTick(event: LorenzTickEvent) {
+        currentSpeed = (Minecraft.getMinecraft().thePlayer.capabilities.getWalkSpeed() * 1000).toInt()
```

Comment:
Use property access syntax instead of the getter method.
```suggestion
        currentSpeed = (Minecraft.getMinecraft().thePlayer.capabilities.walkSpeed * 1000).toInt()
```